06-gitops.md

```diff
@@ -46,13 +50,38 @@ GitOps allows a team to author Kubernetes manifest files, persist them in their
    # If you receive "no" to this command, check which user you authenticated as and ensure they are
    # assigned to the Azure AD Group you designated for cluster admins.
    kubectl auth can-i create namespace -A
-   
+
    kubectl create namespace cluster-baseline-settings
    ```
 
+1. Import cluster management images to your container registry.
+
+   > Responsible cluster operators like to minimize their risk with third party dependencies for container registries. Public container registries are subject to faults such as outages (no SLA) or request throttling. Interruptions like these can be crippling for a system that needs to pull an image _right now_. The AKS service itself depends on Microsoft-hosted container registries. As you bring in third-party images in your cluster, consider where you're storing them. Ideally, you'll depend on your own personal container registry wherever possible. As such, it is recommended to pull in the images that you use into a registry you control, such as the SLA-backed Azure Container Registry deployed with the cluster.
```

Comment:
The AKS service itself depends on Microsoft-hosted container registries.

I find myself re-reading this sentence a few times wondering if it's something that I care about, should be concerned about, influences my decisions. 

Does it add value here?

Reply:
No, not really.  Was just trying to fend off the "but many core images images still come from public repos, just microsoft-controlled ones" feedback.  But if that happens, that's okay, that's a good conversation to have - and honestly this probably wouldn't have prevented it.

Reply:
Resolving as this paragraph was rewritten and excluded this sentence.